apps/mobile/src/app/_layout.tsx

```diff
@@ -43,17 +43,12 @@ import { Box, SheetProvider } from '@leather.io/ui/native';
 
 dayjs.extend(relativeTime);
 
-// Sentry.init({
-//   dsn: process.env.EXPO_PUBLIC_SENTRY_DSN,
-//   environment: process.env.EXPO_PUBLIC_NODE_ENV ?? 'development',
-//   // Adds more context data to events (IP address, cookies, user, etc.)
-//   // For more information, visit: https://docs.sentry.io/platforms/react-native/data-management/data-collected/
-//   sendDefaultPii: true,
-//   integrations: [Sentry.feedbackIntegration()],
-//
-//   // uncomment the line below to enable Spotlight (https://spotlightjs.com)
-//   // spotlight: __DEV__,
-// });
+Sentry.init({
+  dsn: process.env.EXPO_PUBLIC_SENTRY_DSN,
+  debug: __DEV__,
+  tracesSampleRate: 1.0,
```

Comment:
I think 1 is way too high for prod, if I remember correctly Sentry charged based on the rate. We can easily go with 0.1 and adjust from there.